### DISCUSSION TITLE
When I try to reset the password, I get an error

### DISCUSSION THREAD

**Discussion by egorov-vlad** [Dokploy/dokploy#577] [Q&A]
Hi. Today I logged into the server, but I was logged out and does not allow me to log in according to the flogging that I scored. I try to reset and get a connection error. Am I doing something wrong or where is the problem?

![image](https://github.com/user-attachments/assets/897a50c4-44f2-49cd-b872-03adf50bd1d7)

**Comment by Siumauricio** [comment]
You are right is not working, I will fix this in #578

**Comment by Siumauricio** [comment]
Fixed the issue in v0.10.2
